scripts/plot_dataframe.py

```diff
@@ -68,6 +73,12 @@ def plot_analyzed_dataframe(args: Namespace) -> None:
     dataframe = analyze.populate_buy_trend(dataframe)
     dataframe = analyze.populate_sell_trend(dataframe)
 
+    trades = {}
```

Comment:
quickly looking at the doc, i think sqlalchemy's filter.all() returns a list, not a dict - so using `trades = []` would be more appropriate here